include/sentry.h

```diff
@@ -418,6 +418,17 @@ SENTRY_EXPERIMENTAL_API sentry_value_t sentry_value_new_thread(
 SENTRY_EXPERIMENTAL_API sentry_value_t sentry_value_new_stacktrace(
     void **ips, size_t len);
 
+/**
+ * Adds a Stack Trace conforming to the Stack Trace Interface to a value.
+ *
+ * The value needs to be either an exception event, or a thread object.
```

Comment:
An exception _event_ object, or an exception object (not event)? The implementation assumes the latter, which also makes more sense.

Reply:
You are right, it should be 

```The value needs to be either an exception or a thread object.```

Reply:
How do you typically proceed with open conversations? I usually wait until the reviewer thinks the comment has been addressed in a follow-up commit, signalling by resolving.